src/stylus/fablevision/trial-count.styl

```diff
@@ -0,0 +1,25 @@
+.geniblocks.trial-count
+  width: 158px
+  height: 107px
+  background-image: url('../resources/fablevision/huds/Trial_screen.png')
+  background-size: 100% 100%
```

Comment:
`contain` means maintain aspect ratio, leaving empty space if necessary. `100% 100%` means fill the area, stretching the image if necessary. In this case, I'm willing to stretch the image because I could imagine needing a bit more horizontal space and stretching the image probably wouldn't be too bad in this case, which could mean not having to go back to FableVision for a larger image. I'm not aware of any way to have the element auto-sized to the size of the background, given that the size of the background is generally specified relative to the size of the element.